Fix the sshd tests

caphrim007 opened the pull request:
@pjbreaux 

Issues:
Fixes #587

Problem:
The sshd tests have been broken since the introduction (I think) of the
boolean reducing code.

Analysis:
This changes the tests to not use deepcopy and instead just select
two distinct resources from the bigip

Tests:
existing sshd functional tests

caphrim007 commented:
@pjbreaux the errors in this pr are related to the profiles thing that you're in the process of fixing

pjbreaux commented:
Yes, they will likely be taken care of by my PR. We can merge yours first.

pjbreaux commented:
But travis is failing as well.

pjbreaux commented:
Flake8: https://travis-ci.org/F5Networks/f5-common-python/jobs/148992664

caphrim007 commented:
fixes incoming

caphrim007 commented:
@pjbreaux ok travis is fixed. i think the remaining issues are the profiles thing you're handling

pjbreaux commented:
Does the above represent a run against 11.5.4 and 11.6.0?

caphrim007 commented:
11.5.4

caphrim007 commented:
the 11.5.4 is the first set run. if it fails then it doesnt make it to 11.6.0

pjbreaux commented:
Before we merge, can you rebase then revert the skip in test/functional/tm/sys/test_sshd.py. The change I made to skip was:

V11_SUPPORTED = ['11.6.0', '11.6.1']
Changed to...
V11_SUPPORTED = ['11.6.2', '11.6.1']

pjbreaux commented:
@caphrim007: also, we'll wait until we close PR #593 and PR #592

caphrim007 commented:
@pjbreaux does it not work in 11.6.0?

caphrim007 commented:
oh jeez, what did I do to my branch :-P

caphrim007 committed:
Fix the sshd tests

Issues:
Fixes #587

Problem:
The sshd tests have been broken since the introduction (I think) of the
boolean reducing code.

Analysis:
This changes the tests to not use deepcopy and instead just select
two distinct resources from the bigip

Tests:
existing sshd functional tests

pjbreaux commented:
I saw the commits first, I typed out a comment, then I saw your comment.

caphrim007 commented:
@zancas @pjbreaux with skipping that module, these tests now pass. merge when ready
